src/main/java/gov/nasa/podaac/swodlr/security/config/WebSecurityConfig.java

```diff
@@ -1,73 +1,40 @@
 
-import gov.nasa.podaac.swodlr.security.authentication.RedirectingServerOauth2AuthorizationRequestResolver;
-import gov.nasa.podaac.swodlr.security.authentication.client.JweCookieReactiveOauth2AuthorizedClientService;
-import gov.nasa.podaac.swodlr.security.authentication.handlers.RedirectOrMessageAuthenticationSuccessHandler;
-import gov.nasa.podaac.swodlr.security.authentication.handlers.UserBootstrapAuthenticationSuccessHandler;
+import gov.nasa.podaac.swodlr.security.SwodlrSecurityProperties;
+import gov.nasa.podaac.swodlr.security.UserBootstrapWebFilter;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Profile;
 import org.springframework.security.config.annotation.web.reactive.EnableWebFluxSecurity;
+import org.springframework.security.config.web.server.SecurityWebFiltersOrder;
 import org.springframework.security.config.web.server.ServerHttpSecurity;
-import org.springframework.security.oauth2.client.AuthorizedClientServiceReactiveOAuth2AuthorizedClientManager;
-import org.springframework.security.oauth2.client.ReactiveOAuth2AuthorizedClientManager;
-import org.springframework.security.oauth2.client.ReactiveOAuth2AuthorizedClientService;
-import org.springframework.security.oauth2.client.registration.ReactiveClientRegistrationRepository;
 import org.springframework.security.web.server.SecurityWebFilterChain;
-import org.springframework.security.web.server.authentication.DelegatingServerAuthenticationSuccessHandler;
 
 @EnableWebFluxSecurity
 @Profile({"!test"})
 public class WebSecurityConfig {
-  private final ReactiveOAuth2AuthorizedClientManager authorizedClientManager;
-  private final ReactiveOAuth2AuthorizedClientService authorizedClientService;
-
-  @Autowired
-  private UserBootstrapAuthenticationSuccessHandler userBootstrapHandler;
-
   @Autowired
-  private RedirectOrMessageAuthenticationSuccessHandler successHandler;
+  private SwodlrSecurityProperties securityProperties;
 
   @Autowired
-  private RedirectingServerOauth2AuthorizationRequestResolver authorizationRequestResolver;
-
-  public WebSecurityConfig(ReactiveClientRegistrationRepository clientRegistrationRepository) {
-    authorizedClientService = new JweCookieReactiveOauth2AuthorizedClientService();
-    authorizedClientManager = new AuthorizedClientServiceReactiveOAuth2AuthorizedClientManager(
-      clientRegistrationRepository, authorizedClientService
-    );
-  }
+  private UserBootstrapWebFilter userBootstrapWebFilter;
 
   @Bean
   public SecurityWebFilterChain securityWebFilterChain(ServerHttpSecurity http) {
     http
         .cors().and()
         .csrf().disable()
-        .authorizeExchange(authorize -> {
-          authorize
-              .pathMatchers("/config").permitAll()
-              .anyExchange().authenticated();
+        .authorizeExchange((exchange) -> {
+          exchange.pathMatchers("/edl/**").permitAll();
```

Comment:
This totally removes the /config endpoint correct?
Does this still allow for an unauthenticated healthcheck for the ALB?

Reply:
Yes, it does. I'm not sure what we'd want to replace that with for the ALB

Reply:
Created another ticket for ALB concern: https://github.com/podaac/swodlr/issues/90